ignite/cmd/network_request_add_account.go

```diff
@@ -0,0 +1,92 @@
+package ignitecmd
+
+import (
+	"errors"
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+
+	"github.com/ignite/cli/ignite/pkg/cliui"
+	"github.com/ignite/cli/ignite/services/network"
+	"github.com/ignite/cli/ignite/services/network/networkchain"
+)
+
+// NewNetworkRequestAddAccount creates a new command to send add account request
+func NewNetworkRequestAddAccount() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "add-account [launch-id] [address] [coins]",
+		Short: "Send request to add account",
+		RunE:  networkRequestAddAccountHandler,
+		Args:  cobra.RangeArgs(2, 3),
+	}
+
+	flagSetClearCache(c)
+	c.Flags().AddFlagSet(flagNetworkFrom())
+	c.Flags().AddFlagSet(flagSetHome())
+	c.Flags().AddFlagSet(flagSetKeyringBackend())
+	c.Flags().AddFlagSet(flagSetKeyringDir())
+	return c
+}
+
+func networkRequestAddAccountHandler(cmd *cobra.Command, args []string) error {
+	session := cliui.New(cliui.StartSpinner())
+	defer session.End()
+
+	nb, err := newNetworkBuilder(cmd, CollectEvents(session.EventBus()))
+	if err != nil {
+		return err
+	}
+
+	// parse launch ID
+	launchID, err := network.ParseID(args[0])
+	if err != nil {
+		return err
+	}
+
+	address := args[1]
+
+	n, err := nb.Network()
+	if err != nil {
+		return err
+	}
+
+	chainLaunch, err := n.ChainLaunch(cmd.Context(), launchID)
+	if err != nil {
+		return err
+	}
+
+	var networkOptions []networkchain.Option
+
+	c, err := nb.Chain(networkchain.SourceLaunch(chainLaunch), networkOptions...)
```

Comment:
If there are no plans to add options it might simpler just to remove the variable:

```suggestion
	c, err := nb.Chain(networkchain.SourceLaunch(chainLaunch))
```